web-console/knikubevirt/knikubevirt.md

```diff
@@ -0,0 +1,31 @@
+## KNI and KubeVirt Designs
+Access design documentation specific to the KNI and KubeVirt features in OpenShift.
+
+### OpenShift 4.2
+- Cluster Dashboard
+- [VM List](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list)
+- [VM Templates](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-templates/vm-templates)
+- [VM Details](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-details/vm-details)
+
+### Future Designs
+
+#### KNI (Kubernetes Native Infrastructure/OpenShift on Bare Metal)
+- Storage Dashboard
```

Comment:
This should go to 4.2

Reply:
We should review the latest design for the storage dashboard and get that added to the repo when possible.

Reply:
Is there a plan to get the cluster dashboard documented on GH as well @lizsurette?

Reply:
Yes definitely. @andybraren has been making some updates to the design from Summit feedback and then will document in a PR. @andybraren do you think it makes sense for you to put up the Storage Dashboard PR too or maybe @yuvalgalanti ?

Reply:
Definitely planned - just figuring out the timing. Cards and dashboards are pretty interconnected, so there'll probably be 9+ simultaneous PRs to review when that happens, which'll be fun.

Reply:
And I think the Storage Dashboard would be a great PR for @yuvalgalanti to take on once we're ready. 🙂

Reply:
Thanks! Let's try to chunk it up into smaller PRs if possible so we can get it merged easier and not hold up for specific cards that may still be WIP.